Dockerfile

```diff
@@ -36,8 +36,8 @@ COPY . /usr/local/src/ort
 WORKDIR /usr/local/src/ort
 
 # Gradle build.
+ENV GRADLE_HOME=/tmp
```

Comment:
This should be `GRADLE_USER_HOME`, not `GRADLE_HOME`, and it needs to contain the `.gradle` part.

Also, can we just do `GRADLE_USER_HOME=/tmp/.gradle` below (in original line 40) to avoid the creation of an intermediate layer due to `ENV`?